craftbook-bukkit/src/main/java/org/enginehub/craftbook/mechanics/area/Gate.java

```diff
@@ -44,21 +44,12 @@
 import org.enginehub.craftbook.mechanic.CraftBookMechanic;
 import org.enginehub.craftbook.mechanic.MechanicType;
 import org.enginehub.craftbook.mechanic.exception.InvalidMechanismException;
-import org.enginehub.craftbook.util.BlockParser;
-import org.enginehub.craftbook.util.BlockUtil;
-import org.enginehub.craftbook.util.ConfigUtil;
-import org.enginehub.craftbook.util.EventUtil;
-import org.enginehub.craftbook.util.ProtectionUtil;
-import org.enginehub.craftbook.util.SignUtil;
+import org.enginehub.craftbook.util.*;
```

Comment:
Would you be able to please make sure that import style is retained & follows our style guide?